application/src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/router/FilterTransitWhenDirectModeIsEmpty.java

```diff
@@ -53,21 +53,26 @@
 public class FilterTransitWhenDirectModeIsEmpty {
 
   private final StreetMode originalDirectMode;
+  private final boolean pageCursorExists;
 
-  public FilterTransitWhenDirectModeIsEmpty(RequestModes modes) {
-    this(modes.directMode);
+  public FilterTransitWhenDirectModeIsEmpty(RequestModes modes, boolean pageCursorExists) {
+    this(modes.directMode, pageCursorExists);
   }
 
-  public FilterTransitWhenDirectModeIsEmpty(StreetMode originalDirectMode) {
+  public FilterTransitWhenDirectModeIsEmpty(
```

Comment:
You should update this class' javadoc and add some to the public methods as well.

Reply:
I updated the class' javadoc and added documentation to two methods